website/static/js/logFeed.js

```diff
@@ -83,9 +83,13 @@ var Log = function(params) {
         return ret;
     });
 
-    //helper function to strip the leading slash for file or folder in log template
-    self.stripLeadingSlash = function(path){
-        return path.replace(/^\//, '');
+    //helper function to strip the slash for file or folder in log template
+    self.stripSlash = function(path, name){
```

Comment:
@brianjgeiger That's a good point. @chennan47  Are you allowed to upload files with trailing slashes to GitHub, Dropbox, etc.?